charts/tf-controller/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: tf-controller
 description: The Helm chart for Weave GitOps Terraform Controller
 type: application
-version: 0.15.2
-appVersion: "v0.15.2"
+version: 0.15.3
```

Comment:
Please do not dump the version yet. We haven't released v0.15.2.
It's OK if the chart test fails for now.

Reply:
Sure I can, but why not a simple `0.15.x` patch? The only change in the helm chart is the `RUNTIME_NAMESPACE` addition to the branch planner + removed healthz endpoint references from branch planner (as it does not have any http endpoints at all).

Reply:
> Also, the branch planner is a big thing, we cannot use a point release for it.

it was already part of the chart.

Reply:
The bump script did nothing just shows the help of `yq` a few times. Tomorrow I'll check it and bump.